### ISSUE TITLE
Choose an alternative to red text and icons (for colourblind users)

### ISSUE THREAD

**Issue by MJWainwright** [epistemonikos/isoq#200]
We currently use red font and colour for important message, alerts that information is missing, and the exclamation mark icons. These are inappropriate for colourblind users. Consider alternatives. (e.g. change the colour? Enable a setting which when chosen would change this text to an alternative colour from an individual's workspace? Is there a way of enabling the functionality without have to get rid of the colour red which is appropriate for most users?
